### PR TITLE
Adding a note about one time password limitations

### PR DESCRIPTION
**Pull request by jeffreylees**
https://docs-content-staging-pr-8384.herokuapp.com/docs/connections/passwordless

and

https://docs-content-staging-pr-8384.herokuapp.com/docs/connections/passwordless/reference/troubleshoot

Both docs include a new section on the limitations of OTP. There's also a couple of minor adjustments to fix the MFA troubleshooting area.

### CODE REVIEW

**Comment by michaelwallen** [comment]
@jeffreylees This all looks great by me. 👍

File: articles/connections/_otp-limitations.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Only the last one-time password issued will be accepted. Once the latest one is issued, any others are invalidated.
```
Comment (siacomuzzi) [inline articles/connections/_otp-limitations.md]: is `password` a right word for a passwordless connection documentation?

File: articles/connections/passwordless/reference/troubleshoot.md [hunk 2]
```diff
@@ -41,4 +37,8 @@ If a user does not receive the requested email or SMS message, here are some thi
 
 ## What if a user receives multiple authentication emails/SMS messages?
 
-If a user accidentally requests more than one email or SMS message containing magic links or authentication codes, the last **five** codes will be valid. After successful log in using any of the codes/links, **all** of the existing codes/links will be invalidated.
\ No newline at end of file
+If a user accidentally requests more than one email or SMS message containing magic links or authentication codes, the last **five** codes will be valid. After successful log in using any of the codes/links, **all** of the existing codes/links will be invalidated.
```
Comment (siacomuzzi) [inline articles/connections/passwordless/reference/troubleshoot.md]: only last code will be valid

File: articles/connections/_otp-limitations.md (hunk 1 above)
Comment (jeffreylees) [reply articles/connections/_otp-limitations.md]: I mean, by itself, no, but in the context of the whole phrase `one-time password` - that's literally what it is. However, we could reword it to say something like "only the most recently issued code will be accepted" if you like that better?

File: articles/multifactor-authentication/troubleshooting.md [hunk 3]
```diff
@@ -55,13 +55,9 @@ If you attempt to send more than ten SMS messages to your device within one hour
 
 When you exceed your messaging limit, you'll need to wait at least an hour after your request for your first message before requesting another. You will receive an additional attempt after the passage of each additional hour.
 
-## OTP-related issues
+## Rejected Codes
```
Comment (siacomuzzi) [inline articles/multifactor-authentication/troubleshooting.md:58]: just fyi - we didn't update this part in the backend (mfa codes)

Comment (jeffreylees) [reply articles/multifactor-authentication/troubleshooting.md:58]: Yeah, the only reason I touched this doc is that I originally thought it applied to all OTP codes, and then I stumbled on the doubled paragraph and such here and figured I might as well correct it.

File: articles/connections/_otp-limitations.md (hunk 1 above)
Comment (siacomuzzi) [reply articles/connections/_otp-limitations.md]: looks like we are using the `password` word in a lot of places (even in the dashboard), so I will leave the decision to the product team

File: articles/connections/passwordless/reference/troubleshoot.md [hunk 4]
```diff
@@ -41,4 +37,8 @@ If a user does not receive the requested email or SMS message, here are some thi
 
 ## What if a user receives multiple authentication emails/SMS messages?
 
-If a user accidentally requests more than one email or SMS message containing magic links or authentication codes, the last **five** codes will be valid. After successful log in using any of the codes/links, **all** of the existing codes/links will be invalidated.
\ No newline at end of file
+If a user accidentally requests more than one email or SMS message containing magic links or authentication codes, only the most recent code/link will be valid. After successful log in using any of the codes/links, **all** of the existing codes/links will be invalidated.
+
```
Comment (siacomuzzi) [inline articles/connections/passwordless/reference/troubleshoot.md]: > After successful log in using any of the codes/links, **all** of the existing codes/links will be invalidated.

This is not accurate. We are keeping only one code at time (the most recent one), so after successful log in, the only existing code will be invalidated.

Comment (siacomuzzi) [reply articles/connections/passwordless/reference/troubleshoot.md]: same conditions you wrote in `/articles/connections/_otp-limitations.md`, right?

**Review by erjohnson — DISMISSED** [review]

File: articles/connections/_otp-limitations.md (hunk 1 above)
Comment (rkhoriander) [reply articles/connections/_otp-limitations.md]: We call this a "one-time-use code" throughout the rest of the new passwordless docs. Just FYI.

File: articles/connections/passwordless/reference/troubleshoot.md (hunk 4 above)
Comment (jeffreylees) [reply articles/connections/passwordless/reference/troubleshoot.md]: OK, just removed this conversation and clarified it in the limitations one

File: articles/connections/_otp-limitations.md (hunk 1 above)
Comment (jeffreylees) [reply articles/connections/_otp-limitations.md]: I feel like that's a thing the PE team/Product needs to discuss. OTP is an industry-standard term. As far as I know, it hasn't been changed in any existing (not new) docs, or as @siacomuzzi says, in the dashboard.

Comment (jeffreylees) [reply articles/connections/_otp-limitations.md]: Consistency across the board is what we should go for, but IMO it doesn't need to be accomplished via this PR.

**Review by panga — APPROVED** [review]
